### DISCUSSION TITLE
basic auth enabling using kube prometheus stack

### DISCUSSION THREAD

**Discussion by rosh0450** [prometheus-operator/prometheus-operator#6276] [Q&A] [answered]
How to enable basic auth for prometheus using kube prometheus stack helm chart. I mean which variable in values flag should be used??

Thanks in advance.

**Comment by slashpai** [comment]
For helm chart you should ask in https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack

**Reply by rosh0450** [reply, marked as answer]
created an issue 
see [#4199](https://github.com/prometheus-community/helm-charts/issues/4199)
